modulus/launch/logging/mlflow.py

```diff
@@ -82,16 +81,19 @@ def initialize_mlflow(
         Returns MLFlow logging client and active run object
     """
     dist = DistributedManager()
-    if DistributedManager.is_initialized() and dist.distributed:
-        group_name = create_ddp_group_tag(run_name)
-        run_name = f"{run_name}-Process_{dist.rank}"
+    root = True
+    if dist.is_initialized():
+        root = dist.rank == 0
+
+    if not root:  # only one process should be logging to mlflow
```

Comment:
Can just do instead of this if/else:
```
if dist.rank != 0:
    return
```

The is initialized doesn't make full sense there because 83 initializes it regardless.